docs/protocol.md

```diff
@@ -11,6 +11,7 @@
   - [Concatenation for hashing](#concatenation-for-hashing)
 - [JSON RPC (low-level)](#json-rpc-low-level)
   - [Query](#query)
+    - [Greeting a server](#greeting-a-server)
```

Comment:
Given the discussion with @pierluca this should no longer be in the query section I assume?

Reply:
Yes you‘re right, the structure is not ideal here .. 🤔 As I have written above I would put heartbeat somehwere else because it and the other server-server messages are wildely different. And yes you are right, broadcast should probably also be in a separate notification section..? (and similarly in the protocol there should be a notification.json) Or do you think it is fine as it is?

I think I need to quickly chat with pierluca / the other TAs :)

Reply:
In terms of correctness, you're right I think the heartbeats, broadcasts and greet server messages should be in separate notification.json. I think that changing this now would require the subsystems (at least the go backend from my pov) to change some things in the way they implement these. 

What I suggest is keeping it like this for now so that we can start with fixing the lao properly and perhaps opening an issue (if with pierluca and the other TAs you decided that it should indeed be in a notification.json) then do the refactoring in another PR because it will likely break multiple CI jobs. 

If you agree with this, I can still put them apart in the documentation in the meanwhile.

Reply:
Oh and I got your reply a bit delayed.

Yes that sounds reasonable! 😄

Reply:
Putting them in a Notification section would still imply creating a `../protocol/notification/notification.json` file tho right ? Or are you just talking about the documentation in `protocol.md` ? I'm sorry, it's a bit confusing ...

Reply:
Sorry I also got your reply (about getting my reply delayed) ... delayed. Is the PR okay now ?

Reply:
I liked your suggestion of at least separating them in the documentation for now but yes, the PR is fine for me 😄